### DISCUSSION TITLE
Update link for backpack v6 in Release note

### DISCUSSION THREAD

**Discussion by christmex** [Laravel-Backpack/community-forum#510] [General]
Hi guys, this is not that important, but some of you may have lost it when you wanted to access the v6 demo.

I'm really excited every day in the first month, cuz I know there would be something cool happen in backpackforlaravel
so today I just read the [release notes v6](https://backpackforlaravel.com/docs/6.x/release-notes), and found that the URL demo it's not working, so I decided to take a look at crt.sh then I saw the correct one was ``` https://demo-v6.backpackforlaravel.com/ ``` not  ``` http://v6demo.backpackforlaravel.com/ ```

cmiiw 🙏, thx

**Comment by jcastroa87** [comment]
Thanks, @christmex. Soon will be fixed.

I will close this issue now.

Thanks again.
